dvc/api/dvclive.py

```diff
@@ -0,0 +1,14 @@
+import os
+from typing import List
+
+from dvc.exceptions import NotDvcRepoError
+from dvc.repo import Repo
+
+
+def summary(path: str, revs: List[str] = None):
+    try:
+        root = Repo.find_root()
+    except NotDvcRepoError:
+        root = os.getcwd()
+
+    Repo(root_dir=root, uninitialized=True).logs.summarize(path, revs)
```

Comment:
Not sure we need that after introducing `logs` command. Might get deleted later.